ui/src/app/views/workspace-library.tsx

```diff
@@ -0,0 +1,176 @@
+import {Component} from '@angular/core';
+import * as React from 'react';
+
+import {Clickable} from 'app/components/buttons';
+import {Header} from 'app/components/headers';
+import {ClrIcon} from 'app/components/icons';
+import {Spinner} from 'app/components/spinners';
+import {ErrorHandlingService} from 'app/services/error-handling.service';
+import {workspacesApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils';
+import {WorkspacePermissions} from 'app/utils/workspace-permissions';
+import {WorkspaceCard} from 'app/views/workspace-list';
+import {ErrorResponse, Profile} from 'generated/fetch';
+
+const styles = reactStyles({
+  navPanel: {
+    display: 'flex', flexDirection: 'column', backgroundColor: '#E7E6F0',
```

Comment:
Please rebase and use the new color palette, rather than hardcoded hashes. If you don't know what this should be, check with Lou. But probably `colors.light`